FLT/Basic/Reductions.lean

```diff
@@ -259,9 +259,7 @@ private lemma j_pos_aux (a b : ℤ) (hb : b ≠ 0) : 0 < (a + b) ^ 2 - a * b :=
     calc
       0 < a * a + a * b + b * b := ?_
       _ = _ := by ring
-    apply add_pos_of_nonneg_of_pos
-    apply add_nonneg (mul_self_nonneg _) h
-    apply mul_self_pos.mpr hb
+    exact add_pos_of_nonneg_of_pos (add_nonneg (mul_self_nonneg _) h) (mul_self_pos.mpr hb)
   | inr h =>
     rw [sub_pos]
     exact h.trans_le (sq_nonneg _)
```

Comment:
```suggestion
  rify
  calc
    (0 : ℝ) < (a ^ 2 + (a + b) ^ 2 + b ^ 2) / 2 := by positivity
    _ = (a + b) ^ 2 - a * b := by ring
```

Reply:
Done, thanks.

Reply:
Looks like you only golfed the first branch of the proof, while my suggestion was to unify the branches. See https://github.com/ImperialCollegeLondon/FLT/pull/212#discussion_r1835698937